apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -699,6 +705,8 @@ suite('thread_list_ui', function() {
 
       assert.isTrue(firstThreadNode.classList.contains('unread'));
       assert.isTrue(secondThreadNode.classList.contains('unread'));
+      sinon.assert.neverCalledWith(MessageManager.markThreadRead, 5, true);
+      sinon.assert.neverCalledWith(MessageManager.markThreadRead, 6, true);
```

Comment:
Instead of this. Do assertion with 'calledWith' like this.
And pass argument as false.      sinon.assert.calledWith(MessageManager.markThreadRead, 5, false);
   sinon.assert.calledWith(MessageManager.markThreadRead, 6, false);